src/content/documentation/develop/request-the-right-permissions.md

```diff
@@ -48,10 +48,10 @@ Also, if an extension update requires additional permissions the user is prompte
 
 If the user chooses not to approve the permissions and cancels the update, the previous version remains installed and available for use.
 
-These messages have the potential to discourage some users from installing your extension, as the messages might suggest it's doing something “scary”. We provide users with an [explanation of these permissions messages](https://support.mozilla.org/kb/permission-request-messages-explained) and guidance on [how to judge if they're appropriate](https://support.mozilla.org/kb/tips-assessing-safety-extension). However, there are several things you can do to reduce the likelihood of users abandoning the install of your extension because of these messages, these include:
+These messages could discourage some users from installing your extension, as they might suggest it's doing something “scary”. We provide users with an [explanation of these permissions messages](https://support.mozilla.org/kb/permission-request-messages-explained) and guidance on [how to judge if they're appropriate](https://support.mozilla.org/kb/tips-assessing-safety-extension). However, there are several things you can do to reduce the likelihood of users abandoning the installation of your extension because of these messages; these include:
 
 - Make sure you aren't requesting unnecessary permissions.
-- Request permissions at runtime, which enables you to ask for the permissions in context and offer a fall back if the user doesn't grant them.
+- Request permissions at runtime, which enables you to ask for the permissions in context and offer a fallback if the user doesn't grant them.
 - Describe why your extension is requesting its permissions in its AMO description.
 
 Tip: Permission warnings aren't issued [when you load an unpacked extension](/documentation/develop/temporary-installation-in-firefox/). For information on how to see the standard runtime permission flow, see [Test permission requests](/documentation/develop/test-permission-requests/).
```

Comment:
Inconsistent terminology. This is the only place in this document that uses "warning" to refer to the install time permission message show to users.

```suggestion
Tip: Permission messages aren't issued [when you load an unpacked extension](/documentation/develop/temporary-installation-in-firefox/). For information on how to see the standard runtime permission flow, see [Test permission requests](/documentation/develop/test-permission-requests/).
```